docs/GettingStarted.md

```diff
@@ -43,6 +43,7 @@ To contribute, check out the [contribution guidelines][contribution docs] and [d
      - [Grape](#grape)
      - [GraphQL](#graphql)
      - [gRPC](#grpc)
+     - [Http.rb](#http.rb)
```

Comment:
nit: http.rb seems to like their name to not be capitalized: https://github.com/httprb/http#about

Reply:
good catch, updated.